lib/board.rb

```diff
@@ -0,0 +1,92 @@
+class Board
+  attr_reader :cells, :height
+  def initialize(height = 4, width = 4)
+    @height = height
+    @width = width
+    @cells = generate_cells
+  end
+
+  def board_letters
+    ("A"..("A".ord + @height - 1).chr).to_a
+  end
+
+  def board_numbers
+    ("1"..(@width.to_s)).to_a
+  end
+
+  def generate_cells
+    generated_cells = {}
+
+    board_letters.each do |letter|
+      board_numbers.each do |number|
+        generated_cells[letter + number] = Cell.new(letter + number)
+      end
+    end
+    generated_cells
```

Comment:
Only thing I might change is use reduce instead of each on line 20.

Reply:
I changed the generate_cells method to use reduce instead of each.

Reply:
Agreed. Thanks for updating that.